### DISCUSSION TITLE
Django Ninja support for python 3.12.4

### DISCUSSION THREAD

**Discussion by adityaachar01** [vitalik/django-ninja#1228] [General]
Does Django ninja work well with python version 3.12.4 as well? On the website [https://pypi.org/project/django-ninja/](https://pypi.org/project/django-ninja/ ) they have mentioned **Requires: Python >=3.7** 
Can I still work with django ninja with python 3.12.4?
Thanks

**Comment by baseplate-admin** [comment]
Yes

**Reply by adityaachar01** [reply]
Thanks, my bad. I was literally doing a string comparison :( I presumed 3.7 is actually greater than 3.12 !! In my mind I was comparing with 3.1 it seems! :P
